README.md

```diff
@@ -49,25 +49,25 @@ When setting up authentication, the provider will always try to use the key flow
    }
    ```
 
-## Key flow
+### Key flow
 
-    The following instructions assume that you have created a service account and assigned it the necessary permissions, e.g. `project.owner`.
+    The following instructions assume that you have created a service account and assigned the necessary permissions to it, e.g. `project.owner`.
 
 To use the key flow, you need to have a service account key, which must have an RSA key-pair attached to it.
 
-When creating the service account key, a new pair can be created automatically, which will be included in the service account key. This will make it much easier to configure the key flow authentication in the CLI, by just providing the service account key.
+When creating the service account key, a new pair can be created automatically, which will be included in the service account key. This will make it much easier to configure the key flow authentication in the [STACKIT CLI](https://github.com/stackitcloud/stackit-cli/tree/main), by just providing the service account key.
 
-**Optionally**, you can provide your own private key when creating the service account key, which will then require you to also provide it explicitly to the CLI, additionaly to the service account key. Check the STACKIT Knowledge Base for an [example of how to create your own key-pair](https://docs.stackit.cloud/stackit/en/usage-of-the-service-account-keys-in-stackit-175112464.html#UsageoftheserviceaccountkeysinSTACKIT-CreatinganRSAkey-pair).
+**Optionally**, you can provide your own private key when creating the service account key, which will then require you to also provide it explicitly to the [STACKIT CLI](https://github.com/stackitcloud/stackit-cli/tree/main), additionally to the service account key. Check the STACKIT Knowledge Base for an [example of how to create your own key-pair](https://docs.stackit.cloud/stackit/en/usage-of-the-service-account-keys-in-stackit-175112464.html#UsageoftheserviceaccountkeysinSTACKIT-CreatinganRSAkey-pair).

```

Comment:
Oh, I think this should have been `provide it explicitly to the Terraform Provider` instead, not the CLI